### DISCUSSION TITLE
osu!lazer Refuses To Launch

### DISCUSSION THREAD

**Discussion by zarrapha** [ppy/osu#26038] [Ideas]
### Type

Crash to desktop

### Bug description

1. Launch osu!lazer from desktop by clicking its desktop icon.
2. No osu!lazer cookie is present.
3. Task manager shows osu!lazer is clearly launching but disappears after 5~ seconds.

Note: osu!lazer was launching fine months ago. Inability to find a way to troubleshoot issue, as the game refuses to mention any error and just disappears from Task Manager. Logs from Event Viewer were used instead, as I do not know where osu!lazer crash logs are located.

### Version

2023.1222.0

### Logs

[osu-crash (12-22-23).zip](https://github.com/ppy/osu/files/13749999/osu-crash.12-22-23.zip)

**Comment by bdach** [comment]
Error reported is a generic "access violation" (0xc000005) as has been reported multiple times before. Cause is unclear. Best bet is to try updating graphics card drivers.
